Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -407,6 +424,14 @@ public final class LoginToutViewController: UIViewController, MFMailComposeViewC
     self.navigationItem.backBarButtonItem = UIBarButtonItem.back(nil, selector: nil)
   }
 
+  private func pushSetYourPasswordViewController() {
+    let vc = SetYourPasswordViewController.instantiate()
+    vc.delegate = self
+    self.navigationController?.pushViewController(vc, animated: true)
+    self.navigationItem
```

Comment:
So the ticket says to remove the login from the nav bar but I agree that the users shouldn't be "stuck" on this page if they don't want to create a new password...

Reply:
Yea I brought this up last week. I just need to up date the ticket